android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/AuthenticationManager.java

```diff
@@ -269,10 +286,9 @@ public Completable signUp(@NonNull final SignUp signUp) {
      * Signs up just using a phone number. No username/password will be stored at this time.
      * <p>
      *
-     * @param phone  participant's phone number.
-     *               To prevent accidental improper settings, study will be set to the studyId
-     *               provided in the Bridge, consent will be set to false, and account status
-     *               will be cleared.
+     * @param phone
+     *         participant's phone number. To prevent accidental improper settings, study will be set to the studyId
+     *         provided in the Bridge, consent will be set to false, and account status will be cleared.
```

Comment:
As we move towards iOS parity, I'd suggest deprecating old method names.  For instance, on iOS...

signUp = signUpStudyParticipant
signIn = signInWithEmail, signInWithExternalID, signInWithCredential, etc.